server/personality/personality.controller.ts

```diff
@@ -136,19 +136,26 @@ export class PersonalityController {
             {
                 language: req.language,
                 order: "random",
-                pageSize: 6,
+                pageSize: 7,
                 fetchOnly: true,
```

Comment:
Suggestion: I would try another way, adding a filter to the query object

```js
_id: { $ne: personality?._id },
```

and in the `personality.service`

```js
verifyInputsQuery(query) {
        const queryInputs: any = {};
        if (query.name) {
            queryInputs.name = { $regex: query.name, $options: "i" };
        }
        if(query._id) {
            queryInputs._id = query._id
        }
        return queryInputs;
    }
```